tools/bug-report/pkg/cluster/cluster.go

```diff
@@ -230,6 +231,12 @@ func GetClusterResources(ctx context.Context, clientset *kubernetes.Clientset, c
 	}
 
 	for i, p := range pods.Items {
+		if p.Namespace == "kube-system" {
```

Comment:
CNI can be in other namespaces (istio-system) is also common, maybe we do not check the namespace?